include/base/dof_map.h

```diff
@@ -719,7 +719,7 @@ class DofMap : public ReferenceCountedObject<DofMap>,
    * \returns The number of degrees of freedom on each partition for a
    * particular variable \p vn.
    */
-  std::vector<dof_id_type> n_dofs_on_each_processor(const unsigned int vn) const
```

Comment:
What's the reason for this?  I found the old name just as understandable, and even if it wasn't we wouldn't want to instantly break the API compatibility with it with no transition period for the migration.

Reply:
The reason is simple, but weak: the name is long and it was taking too much of my lines of code, that's all.
Since I introduced the function two months ago and I couldn't see no usage otherwise, I thought I might be in the grace period. Happy to drop.

Reply:
Oh, wow, Nov. 6 commit, merged even later than that?  Never mind, then, I'm cool with the change.  Anyone keeping up with libMesh *that* closely can just keep doing so.  ;-)